ssl.go

```diff
@@ -105,16 +109,15 @@ func sslClientCertificates(tlsConf *tls.Config, o values) {
 
 // sslCertificateAuthority adds the RootCA specified in the "sslrootcert" setting.
 func sslCertificateAuthority(tlsConf *tls.Config, o values) {
-	if sslrootcert := o.Get("sslrootcert"); sslrootcert != "" {
+	if sslrootcert, ok := o["sslrootcert"]; ok {
```

Comment:
I think you're right, but as mentioned above, I think this is OK until we get a report of someone relying on this behaviour. WDYT?